### DISCUSSION TITLE
Some content on channel 3 instead of 1+2

### DISCUSSION THREAD

**Discussion by 2021-lex** [ExistentialAudio/BlackHole#538] [Q&A]
Hi! Some content, for example this youtube video;
https://www.youtube.com/watch?v=_6F9bC4TpIU
The sound comes on channel 3 instead of 1+2 as expected...
BlackHole 16ch

Any ideas why?

**Comment by devinroth** [comment]
Chances are you’re using an aggregate device.

**Reply by 2021-lex** [reply]
Hi! thansk for the qick responce 👍 
I don't think aggregated device is the problem.
Most content work just fine. but this video behaves differently, I think the problem is that the sound is in mono. can you have a look and se if you experience the same problem?

**Comment by devinroth** [comment]
It’s the aggregate I guarantee.

Tell me exactly how you have it setup and I’ll explain.

**Comment by 2021-lex** [comment]
System soundcard Blackhole_16ch
Element vst host https://kushview.net/
Audio Input Device: Blackhole_16ch
Audio Output Device: External headphones

<img width="696" alt="Screenshot 2022-02-15 at 16 17 33" src="https://user-images.githubusercontent.com/80168213/154091794-3c589eee-7de8-4155-a26a-a23bc8429b62.png">

In that clip I linked to: https://www.youtube.com/watch?v=_6F9bC4TpIU 
sound is from channel 3. I think it is because of mono sound.

**Comment by devinroth** [comment]
But how are you sending audio to BlackHole?

**Reply by 2021-lex** [reply]
BlackHole is the system soundcard, so all sound on the computer go to BlackHole and in Element from BlackHole to external Headphones.

**Comment by devinroth** [comment]
And how are you recording it?

It’s literally impossible that BlackHole is changing channels FYI.

**Comment by 2021-lex** [comment]
I think I know what the problem is!
if I have 7.1 config selected in Audio Midi Setup under Configure Speakers... 
CoreAudio sends mono on center speaker (channel 3) if it is a multichannel setup.
If I select Stereo instead it sends mono on LR channel.

Thanks for your help! 👍
